cv_bridge/CMakeLists.txt

```diff
@@ -3,16 +3,18 @@ project(cv_bridge)
 
 find_package(catkin REQUIRED COMPONENTS rosconsole sensor_msgs)
 
+find_package(Boost QUIET)
```

Comment:
I guess this could be moved one line down, i.e. inside the `if` branch. Probably doesn't matter much.

Reply:
Oh shoot, it should be. I'm assuming the android environment doesn't have/work with boost. Will fix.

Reply:
Oh I guess would have been no harm, but certainly unnecessary to find twice. Fixed.